sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/intervalExpressions.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.expressions.interval
```

Comment:
`org.apache.spark.sql.catalyst.expressions.interval` -> `org.apache.spark.sql.catalyst.expressions`?

Reply:
Actually, `package` name should be consistent with the directory structure.

Reply:
I thought about keeping the interval expression classes in `org.apache.spark.sql.catalyst.expressions` but in that case I should add some prefix to all classes to distinguish them from date-time expression classes, probably. Semantically, this is the same as moving them to a separate sub-package, isn't it? ;-)

Reply:
Yep. I understood the intention.